test/stress/generate-report/generate_report.py

```
@@ -0,0 +1,860 @@
+                quantile_cont(podCreatedMs, 0.9) as podCreated_p90,
+                quantile_cont(podScheduledMs, 0.5) as podScheduled_p50,
+                quantile_cont(podScheduledMs, 0.9) as podScheduled_p90,
+                quantile_cont(podRunningMs, 0.5) as podRunning_p50,
+                quantile_cont(podRunningMs, 0.9) as podRunning_p90,
+                quantile_cont(sandboxReadyMs, 0.5) as sandboxReady_p50,
+                quantile_cont(sandboxReadyMs, 0.9) as sandboxReady_p90,
+                quantile_cont(sandboxReadyMs, 0.99) as sandboxReady_p99
+            FROM read_json_auto('{str(sandboxes_file)}')
+            GROUP BY phase
+            ORDER BY phase
+        """).fetchall()
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)